netebpfext/user/fwp_um.cpp

```diff
@@ -796,6 +808,10 @@ _IRQL_requires_max_(DISPATCH_LEVEL) NTSTATUS NTAPI FwpsAcquireWritableLayerDataP
     _Out_ void** writableLayerData,
     _Inout_opt_ FWPS_CLASSIFY_OUT0* classifyOut)
 {
+    if (ebpf_fault_injection_inject_fault()) {
+        return STATUS_NO_MEMORY;
```

Comment:
Does WFP actually do a memory allocation in this API?  
https://learn.microsoft.com/en-us/windows-hardware/drivers/ddi/fwpsk/nf-fwpsk-fwpsacquirewritablelayerdatapointer0 doesn't mention STATUS_NO_MEMORY, should it?

Reply:
Yes, From the source code for this API definition, there is allocation happening
 WfpPoolAllocNonPaged